components/Typography/typography.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Czar ui</title>
+
+    <!-- custom css -->
+    <link rel="stylesheet" href="../../app.css" />
+
+    <!-- font awesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+      integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="container m-auto">
+      <h1 class="text-brand">Czar ui</h1>
+
+      <!-- Typography Starts -->
+
+      <section class="section-container">
+        <h2 class="section-head">Typography - Alignment</h2>
+
+        <p class="text-left">Left Alignment</p>
+        <p class="text-center">Center Alignment</p>
+        <p class="text-right">Right Alignment</p>
+      </section>
+
+      <section class="section-container">
+        <h2 class="section-head">Typography - Display Style</h2>
+
+        <p class="fs-xs">Extra Small - XS</p>
+        <p class="fs-sm">Small - SM</p>
+        <p class="fs-md">Medium -MD</p>
+        <p class="fs-lg">Large - LG</p>
+        <p class="fs-xl">Extra Large - XL</p>
+        <p class="fs-xxl">Extra Extra Large - XXL</p>
+      </section>
+
+      <section class="section-container">
+        <h2 class="section-head">Typography - Headings</h2>
+
+        <h1 class="my-0p5">h1.heading</h1>
+        <h2 class="my-0p5">h2.heading</h2>
```

Comment:
on github box H1 and H2 are appearing of same size , please correct this. Rest all is good!